.husky/post-checkout

```diff
@@ -5,4 +5,5 @@
 npx --no-install yarnhook
 
 # make sure the autoload files are regenerated
+composer install
```

Comment:
@marcinbot I'm not too familiar with husky but if I understand what it is, I think it's safe to run `composer install` here, right?

Reply:
Thanks. Addressed in a224da1e8ac01ebcfb7b758cf400387344e461eb.